### DISCUSSION TITLE
trying to use bridge

### DISCUSSION THREAD

**Discussion by tiagonux** [srl-labs/containerlab#409] [Q&A] [answered]
Hi guys,

I have started to test the bridge(for STP lab) and previously I created the bridge on the OS(CentOS 8) as well.
But I got the following error during the deploy:

[root@lab3-container-lab-1 lab3]# containerlab deploy -t stp1.yml 
INFO[0000] Parsing & checking topology file: stp1.yml   
INFO[0000] Creating lab directory: /root/lab3/clab-stp  
INFO[0000] Creating docker network: Name='clab', IPv4Subnet='172.20.20.0/24', IPv6Subnet='2001:172:20:20::/64', MTU='1500' 
INFO[0000] Creating container: sw1-a                    
INFO[0000] Creating container: sw1-b                    
INFO[0000] Restarting 'sw1-a' node                      
INFO[0000] Restarting 'sw1-b' node                      
INFO[0001] Creating virtual wire: sw1-a:eth1 <--> br-sw1-mlag:vnet1 
INFO[0001] Creating virtual wire: sw1-b:eth1 <--> br-sw1-mlag:vnet2 
ERRO[0001] could not lookup "sw1-b": Link not found     
ERRO[0001] could not lookup "sw1-a": Link not found     
INFO[0001] Writing /etc/hosts file         

From "ip a" output I can see my current bridge:

5: br-sw1-mlag: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9216 qdisc noqueue state UNKNOWN group default qlen 1000
    link/ether 66:ad:ef:46:a2:92 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::64ad:efff:fe46:a292/64 scope link 
       valid_lft forever preferred_lft forever

On the my setup I have this topology: 

name: stp 

topology:
  nodes:
    sw1-a:
      kind: ceos
      image: ceos:4.26.0F
    sw1-b:
      kind: ceos
      image: ceos:4.26.0F
    br-sw1-mlag:
      kind: bridge

  links:
      - endpoints: ["sw1-a:eth1", "br-sw1-mlag:vnet1"]
     - endpoints: ["sw1-b:eth1", "br-sw1-mlag:vnet2"]

Let me know if I'm doing something wrong.

Thanks

Tiago Pires

**Comment by hellt** [comment, marked as answer]
Hey,
sorry that was a regression in 0.13.0 version,
check this post with a workaround until 0.14.0 is ready - https://github.com/srl-labs/containerlab/issues/399

**Reply by tiagonux** [reply]
Oh, I did not see that. I fixed it and now is working. 
Thank you.
